docs/snapshotting/snapshot-support.md

```diff
@@ -25,6 +25,8 @@
   - [Secure and insecure usage examples](#usage-examples)
   - [Reusing snapshotted states securely](#reusing-snapshotted-states-securely)
 - [Vsock device limitation](#vsock-device-limitation)
+- [VMGenID device limitation](#vmgenid-device-limitation)
+- [Where can I resume my snapshots?](#where-can-i-resume-my-snapshots)
```

Comment:
(out of scope) can we autogenerate those somehow?

Reply:
Maybe we can do something like that as part of addressing #5009? We could definitely have a test that check we're not missing any entries here though.